oak_functions/examples/hello_world/config.toml

```diff
@@ -1,3 +1,5 @@
 # TODO(#1973): Change schema to HTTPS when supported.
 lookup_data_url = "http://storage.googleapis.com/oak_lookup_data/lookup_data_weather"
 lookup_data_download_period = "10m"
+policy.constant_response_size_bytes = 90
+policy.constant_processing_time = 20ms
```

Comment:
I think this needs to be quoted? Even github seems to think this is not syntactically valid (perhaps the Rust parser is more lenient).